README.md

```diff
@@ -69,8 +69,7 @@ Or, using a script tag in the browser (only works in some browsers, see list bel
 ffmpeg.wasm provides simple to use APIs, to transcode a video you only need few lines of code:
 
 ```javascript
-const fs = require('fs');
-const { createFFmpeg, fetchFile } = require('@ffmpeg/ffmpeg');
+import { createFFmpeg, fetchFile } from '@ffmpeg/ffmpeg';
```

Comment:
I think we still need to import `fs` here

Reply:
Good catch - hopefully the named import I've added works for you